pkgs/tools/misc/lerpn/default.nix

```
@@ -0,0 +1,28 @@
+{ python3, lib, fetchFromGitea }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "lerpn";
+  version = "unstable-2023-06-09";
+
+  src = fetchFromGitea {
+    domain = "gitea.alexisvl.rocks";
+    owner = "alexisvl";
+    repo = pname;
```

```suggestion
    repo = "lerpn";
```

No reason to do this besides overusing `rec`.